module/VuFind/src/VuFind/Controller/Plugin/Favorites.php

```diff
@@ -193,10 +197,10 @@ public function delete($ids, $listID, $user)
                 $user->removeResourcesById($ids, $source);
             }
         } else {
-            $table = $this->getController()->getTable('UserList');
-            $list = $table->getExisting($listID);
+            $listService = $this->favoritesService->getDbService(\VuFind\Db\Service\UserListService::class);
```

Comment:
It would be more consistent with the original code to use `$this->getController()` here instead of `$this->favoritesService`. The `getDbService` method is available in both places, but since this is a controller plugin, pulling the dependency from the controller seems slightly more appropriate.